nautobot/core/tests/nautobot_config.py

```diff
@@ -26,9 +26,19 @@
 
 # Redis variables
 
+_GROUP_INDEX = os.getenv("NAUTOBOT_TEST_GROUP_INDEX", "")
+if _GROUP_INDEX:
+    _redis_index = int((int(_GROUP_INDEX) + 2) * 2)
```

Comment:
Please add a comment explaining this calculation?

Reply:
Fixed